cmd/bpf2go/main.go

```diff
@@ -75,6 +75,49 @@ func run(stdout io.Writer, pkg, outputDir string, args []string) (err error) {
 		outputDir: outputDir,
 	}
 
+	err = b2g.parseArgs(args)
+	switch {
+	case err == nil:
+		return b2g.convertAll()
+	case errors.Is(err, flag.ErrHelp):
+		return nil
+	default:
+		return err
+	}
+}
+
+type bpf2go struct {
+	stdout io.Writer
+	// Absolute path to a .c file.
+	sourceFile string
+	// Absolute path to a directory where .go are written
+	outputDir string
+	// Alternative output stem. If empty, identStem is used.
+	outputStem string
+	// Valid go package name.
+	pkg string
+	// Valid go identifier.
+	identStem string
+	// Targets to build for.
+	targetArches map[target][]string
+	// C compiler.
+	cc string
+	// Command used to strip DWARF.
+	strip            string
+	disableStripping bool
+	// C flags passed to the compiler.
+	cFlags          []string
+	skipGlobalTypes bool
+	// C types to include in the generatd output.
+	cTypes cTypes
+	// Build tags to be included in the output.
+	tags buildTags
+	// Base directory of the Makefile. Enables outputting make-style dependencies
+	// in .d files.
+	makeBase string
+}
+
+func (b2g *bpf2go) parseArgs(args []string) (err error) {
```

Comment:
Nit: my preference is to have constructor like function as much as possible. How would you feel about `newB2G(stdout io.Writer, pkg, outputDir string, args []string) (*b2g, error)`?

Reply:
Done.